app/assets/javascripts/discourse/app/components/composer-editor.js

```diff
@@ -677,6 +677,35 @@ export default Component.extend(ComposerUpload, {
     return;
   },
 
+  resetImageControls: function (buttonWrapper) {
```

Comment:
I think we usually prefer this:

```suggestion
  resetImageControls(buttonWrapper) {
```